stablehlo/dialect/StablehloOps.cpp

```diff
@@ -518,6 +518,16 @@ mlir::Attribute CustomCallOp::getBackendConfigOrDefault() {
   return StringAttr::get(getContext(), "");
 }
 
+// Returns if the backend config is unset, or if empty dict / string attribute.
+bool CustomCallOp::hasEmptyBackendConfig() {
+  if (!getBackendConfig().has_value()) return true;
```

Comment:
Thanks for taking care of the change. 

nit:  we can remove the line `if (!getBackendConfig().has_value()) return true;` to make API simpler and  call  `getBackendConfigOrDefault` first..

Reply:
Good point! I'm thinking to leave this as-is, the `OrDefault` method will construct an object if its empty, so in this current state its more efficient and doesn't have that null-attr overhead